src/redux/actions/projects/createProject.js

```diff
@@ -0,0 +1,28 @@
+import moment from 'moment';
+import hash from 'object-hash';
+
+import {
+  PROJECTS_CREATE,
+} from '../../actionTypes/projects';
+import { projectTemplate } from '../../reducers/projects/initialState';
+
+const createProject = (
+  projectName,
+) => async (dispatch) => {
+  const createdAt = moment().local().format('DD MMM YYYY, HH:mm:ss [GMT]Z');
+
+  const newProject = {
+    ...projectTemplate,
+    name: projectName,
+    uuid: hash(projectName + moment().format('DDMMYYY')).slice(0, 10),
```

Comment:
take only the first 10 character of generated hash, otherwise it becomes too long and unwieldy. Thoughts?

Reply:
My favorite way to make cute random IDs:

https://github.com/a-type/adjective-adjective-animal

Reply:
Avoding clashes is the priority. I'll change it to using UUID.